src/problems/unconstrain.cpp

```diff
@@ -167,7 +167,7 @@ vector_double unconstrain::fitness(const vector_double &x) const
                 }
             }
             for (decltype(nc) i = 0u; i < nc; ++i) {
-                if (c[i] > 0.) {
+                if (!(c[i] < 0.)) {
```

Comment:
Shouldn't this be ``<=`` rather than ``<`` to match the original code for non-NaN values?

Reply:
True! It leads to the same behavior in practice, since it only differs for c[i] == 0, where then 0 is added to the penalty. Still, pushed a change to make it consistent.